### ISSUE TITLE
No alternative to two finger scrolling - WCAG 2.5.1 failure

### ISSUE THREAD

**Issue by RichardDavies** [elmarquis/Leaflet.GestureHandling#102]
This plugin requires users to use two fingers to scroll or pan the map on a mobile device, which is an accessibility issue. [WCAG 2.5.1 requires that functionality be operable with a single finger pointer gesture for those users with limited physical abilities](https://www.w3.org/WAI/WCAG22/Understanding/pointer-gestures.html). There is no alternative way to move the map or means whereby a user can disable this two-finger requirement.

### Possible Solutions

- Provide an alternative way for users to move the map, such as [arrow buttons that can be activated with a single finger](https://github.com/kartena/Leaflet.Pancontrol). 
- Provide a mechanism for users to disable or turn off the two-finger gesture requirement.
